### PR TITLE
BigBlueButton 2.4 Add option to disable/enable the Learning Dashboard Feature

### PR DESCRIPTION
**Pull request by BrutalBirdie**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -151,6 +151,7 @@ bbb_breakout_rooms_privatechat_enabled: true
 bbb_user_inactivity_inspect_timer: 0
 bbb_user_inactivity_threshold: 30
 bbb_user_activity_sign_response_delay: 5
+bbb_learningdashboardenabled: true
```
Comment (EdJoPaTo) [inline defaults/main.yml:154]: (I saw camelCase not working.) Maybe use dashes like with the other variables to improve readability?
`bbb_learning_dashboard_enabled`

File: templates/bbb-web.properties [hunk 2]
```diff
@@ -37,6 +37,7 @@ breakoutRoomsPrivateChatEnabled={{ bbb_breakout_rooms_privatechat_enabled | tern
 userInactivityInspectTimerInMinutes={{ bbb_user_inactivity_inspect_timer }}
 userInactivityThresholdInMinutes={{ bbb_user_inactivity_threshold }}
 userActivitySignResponseDelayInMinutes={{ bbb_user_activity_sign_response_delay }}
+learningDashboardEnabled={{ bbb_learningDashboardEnabled }}
```
Comment (EdJoPaTo) [inline templates/bbb-web.properties:40]: Variable has wrong casing here after renaming. (consider my other comment first)
```suggestion
learningDashboardEnabled={{ bbb_learningdashboardenabled }}
```

Comment (BrutalBirdie) [reply templates/bbb-web.properties:40]: good catch.
Both done.

**Review by crosscodr — APPROVED** [review]

File: templates/bbb-web.properties (hunk 2 above)
Comment (Davka) [reply templates/bbb-web.properties]: I would only set this option if you are on BBB 2.4.

**Comment by BrutalBirdie** [comment]
Lets not merge this until we are going BBB 2.4

**Review by Davka — APPROVED** [review]

**Pull request merged**